components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/servlet/CommonAuthenticationServlet.java

```diff
@@ -44,17 +44,16 @@ public void init() {
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
-        doPost(request, response);
+        CommonAuthenticationHandler commonAuthenticationHandler = new CommonAuthenticationHandler();
+        commonAuthenticationHandler.doGet(request, response);
     }
 
     @Override
     protected void doPost(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
 
-        if (FrameworkUtils.getMaxInactiveInterval() == 0) {
-            FrameworkUtils.setMaxInactiveInterval(request.getSession().getMaxInactiveInterval());
-        }
-        FrameworkUtils.getRequestCoordinator().handle(request, response);
+        CommonAuthenticationHandler commonAuthenticationHandler = new CommonAuthenticationHandler();
```

Comment:
let's make this commonAuthenticationHandler as class level variable

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/4509/commits/fbb5f9065181ab04c773a345028bfe9ebdba916f